diff -up wpa_supplicant-0.6.8/wpa_supplicant/Makefile.qt4 wpa_supplicant-0.6.8/wpa_supplicant/Makefile
--- wpa_supplicant-0.6.8/wpa_supplicant/Makefile.qt4	2010-05-11 16:12:15.657510359 -0700
+++ wpa_supplicant-0.6.8/wpa_supplicant/Makefile	2010-05-11 16:12:25.646383528 -0700
@@ -1165,7 +1165,7 @@ wpa_gui: wpa_gui/Makefile
 	$(MAKE) -C wpa_gui
 
 wpa_gui-qt4/Makefile:
-	qmake -o wpa_gui-qt4/Makefile wpa_gui-qt4/wpa_gui.pro 
+	qmake-qt4 -o wpa_gui-qt4/Makefile wpa_gui-qt4/wpa_gui.pro 
 
 wpa_gui-qt4: wpa_gui-qt4/Makefile
 	$(MAKE) -C wpa_gui-qt4
